From khali@linux-fr.org Wed Oct 26 12:20:05 2005
Date: Wed, 26 Oct 2005 21:20:17 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 07/16] i2c: Drop unused parport i2c IDs
Message-Id: <20051026212017.23a70d41.khali@linux-fr.org>
Content-disposition: inline; filename=i2c-id-drop-old-parport.patch

Drop unused i2c-over-parallel-port i2c IDs:
* I2C_HW_B_LPC was never actually used as far as I could search.
* I2C_HW_B_ELV and I2C_HW_B_VELLE are no more used since the
  introduction of the unified i2c-parport driver in Linux 2.6.2.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/i2c-id.h |    3 ---
 1 file changed, 3 deletions(-)

--- gregkh-2.6.orig/include/linux/i2c-id.h
+++ gregkh-2.6/include/linux/i2c-id.h
@@ -164,10 +164,7 @@
 
 /* --- Bit algorithm adapters 						*/
 #define I2C_HW_B_LP		0x010000 /* Parallel port Philips style */
-#define I2C_HW_B_LPC		0x010001 /* Parallel port control reg. */
 #define I2C_HW_B_SER		0x010002 /* Serial line interface */
-#define I2C_HW_B_ELV		0x010003 /* ELV Card */
-#define I2C_HW_B_VELLE		0x010004 /* Vellemann K8000 */
 #define I2C_HW_B_BT848		0x010005 /* BT848 video boards */
 #define I2C_HW_B_WNV		0x010006 /* Winnov Videums */
 #define I2C_HW_B_VIA		0x010007 /* Via vt82c586b */
